opensrp-chw/src/nacp/assets/json.form/hivst_mobilization_session.json

```diff
@@ -51,13 +51,13 @@
         "openmrs_entity": "concept",
         "openmrs_entity_id": "mobilization_date",
         "type": "date_picker",
-        "hint": "Mobilization Date",
+        "hint": "Tarehe ya uhamasishaji",
         "expanded": false,
         "min_date": "today-6m",
         "max_date": "today",
         "v_required": {
           "value": "true",
-          "err": "Please enter the mobilization date"
+          "err": "Tafadhali ingiza tarehe ya uhamasishaji"
         }
       },
       {
```

Comment:
This needs to be in opensrp-chw/src/nacp/assets/json.form-sw/hivst_mobilization_session.json